package.json

```
@@ -29,6 +29,7 @@
     "@cypress/eslint-plugin-json": "3.2.1",
     "condition-circle": "2.0.2",
     "eslint": "^5.7.0",
+    "eslint-plugin-eslint-plugin": "^3.6.1",
```

```suggestion
    "eslint-plugin-eslint-plugin": "^4.0.1",
```

Can you bump to v4 which was just released today? 

https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/releases/tag/v4.0.0